### PR TITLE
Introduces yard

### PR DESCRIPTION
**Pull request by christinach**
Adds yard gem, Adds yardoc and doc in the .gitignore
Introduces updates for ElectronicLocation and AlmaAdapter::Bib

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Liberate MARC data from Voyager.
 
+[![CircleCI](https://circleci.com/gh/pulibrary/marc_liberation.svg?style=svg)](https://circleci.com/gh/pulibrary/marc_liberation)
+[![CoverageStatus](https://coveralls.io/repos/github/pulibrary/marc_liberation/badge.svg?branch=main)](https://coveralls.io/github/pulibrary/marc_liberation?branch=main)
+[![InchCI](https://inch-ci.org/github/pulibrary/marc_liberation.svg?branch=main)](https://inch-ci.org/github/pulibrary/marc_liberation)
 ## Services
```
Comment (christinach) [inline README.md:8]: I can't see the docs badge after changing master to main.

Comment (jrgriffiniii) [reply README.md:8]: There may be a delay on the servers for inch-ci.org.

Comment (christinach) [reply README.md:8]: when I https://inch-ci.org/github/pulibrary/marc_liberation?pending_build=607563 it never builds. maybe I need to configure something more?

File: .circleci/config.yml [hunk 2]
```diff
@@ -80,6 +80,9 @@ jobs:
             TRAJECT_CONFIG: marc_to_solr/lib/traject_config.rb
             BIBDATA_ADMIN_NETIDS: admin123
             HATHI_OUTPUT_DIR: marc_to_solr/spec/fixtures/
+      - run:
+          name: Build the YARD documentation
+          command: bundle exec yardoc
```
Comment (jrgriffiniii) [inline .circleci/config.yml:85]: One may also preserve the generated documentation using:
```
- store_artifacts:
          path: ~/marc_liberation/doc
```

File: README.md (hunk 1 above)
Comment (christinach) [reply README.md:8]: I meant to write when I try to build the branch name: https://inch-ci.org/github/pulibrary/marc_liberation?branch=693-yard&pending_build=607564

**Review by jrgriffiniii — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (christinach) [reply README.md:8]: it is interesting that I can see https://inch-ci.org/github/pulibrary/marc_liberation?branch=master but not  https://inch-ci.org/github/pulibrary/marc_liberation?branch=main.

**Comment by jrgriffiniii** [comment]
I am afraid that InchCI seems to be breaking for me still:

image

**Comment by christinach** [comment]
> I am afraid that InchCI seems to be breaking for me still:
> 
> image

yes I can see that too when I use the branch name or the main .

**Review by jrgriffiniii — APPROVED** [review]

**Comment by christinach** [comment]
https://inch-ci.org/github/pulibrary/marc_liberation?branch=main

**Review by jrgriffiniii — APPROVED** [review]

**Pull request merged**